pkg/agent/cniserver/server.go

```diff
@@ -497,6 +497,11 @@ func (s *CNIServer) CmdCheck(_ context.Context, request *cnipb.CniCmdRequest) (
 	return &cnipb.CniCmdResponse{CniResult: []byte("")}, nil
 }
 
+// getContainerKey returns the key of a Pod, which is a string with format "$K8S_POD_NAMESPACE/$K8S_POD_NAME".
+func (c *CNIConfig) getContainerKey() string {
+	return strings.Join([]string{string(c.K8S_POD_NAMESPACE), string(c.K8S_POD_NAME)}, "/")
```

Comment:
Nit - Sprintf() or just c.K8S_POD_NAMESPACE + "/” + c.K8S_POD_NAME might be more efficient.

Reply:
Sure, will update.